embeddings/evaluator/submission.py

```diff
@@ -28,6 +28,58 @@ class Submission:
     packages: List[str]
     config: Optional[Dict[str, Any]] = None  # any additional config
 
+    @staticmethod
+    def _init_evaluator_kwargs(hparams: Dict[str, Any]) -> Dict[str, Any]:
+        evaluator_kwargs = {}
+        evaluation_mode = hparams.get("evaluation_mode", None)
+        tagging_scheme = hparams.get("tagging_scheme", None)
+
+        if evaluation_mode or tagging_scheme:
+            evaluator_kwargs.update(
+                {
+                    "evaluation_mode": evaluation_mode,
+                    "tagging_scheme": tagging_scheme,
+                }
+            )
+        return evaluator_kwargs
+
+    @staticmethod
+    def from_local_disk(
+        submission_name: str,
+        evaluation_file_path: T_path,
+        packages_file_path: T_path,
+        wandb_log_dir: T_path,
+        best_params_path: T_path,
+        task: str,
+    ) -> "Submission":
+        [wandb_run_dir] = [it for it in Path(wandb_log_dir).iterdir() if "run" in str(it)]
+        wandb_config_path = wandb_run_dir / "files" / "config.yaml"
+
+        with wandb_config_path.open() as f:
+            wandb_cfg = yaml.load(f, Loader=yaml.Loader)
+        with Path(evaluation_file_path).open() as f:
+            evaluation_json = f.read()
+        with Path(best_params_path).open() as f:
+            hparams = yaml.load(f, Loader=yaml.Loader)
```

Comment:
I believe we have `read_yaml` and probably also `read_json` functions defined in our project - we can use them :)

Reply:
Our `read_yaml` function doesn't handle this case. We don't have 'read_json` function.